### PR TITLE
logging operator

### PR DESCRIPTION
**Pull request by dian-xh**
logging operator pr:

https://github.com/IBM/ibm-elastic-stack-operator/pull/2

### CODE REVIEW

File: deploy/crds/operator.ibm.com_v1alpha1_operandconfig_cr.yaml [hunk 1]
```diff
@@ -71,3 +71,193 @@ spec:
   - name: ibm-monitoring-grafana-operator
     spec:
       grafana: {}
+  - name: ibm-elastic-stack-operator
+    spec:
+#      elasticStack: {}
+      elasticStack:
```
Comment (chenzhiwei) [inline deploy/crds/operator.ibm.com_v1alpha1_operandconfig_cr.yaml]: please remove the parameters in this CR.

This OperandConfig CR is managed by administrators and only the administrator faced parameters can be added here.

**Comment by dian-xh** [comment]
assign @chenzhiwei

**Comment by dian-xh** [comment]
hi @chenzhiwei , changes are made so that CR parameters moved into the csv. please review. thanks!

**Comment by horis233** [comment]
@dian-xh Please run `make code-dev` to update CSV for the change.

**Comment by dian-xh** [comment]
done:
```
INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [operator:[v1alpha1], ] 
INFO[0028] Code-generation complete.                    
Updating the CRD files with the OpenAPI validations
operator-sdk generate crds
INFO[0000] Running CRD generator.                       
INFO[0001] CRD generation complete.                     
Updating/Generating a ClusterServiceVersion YAML manifest for the operator
operator-sdk generate csv --csv-version 0.0.1 --update-crds
INFO[0000] Generating CSV manifest version 0.0.1        
INFO[0084] Fill in the following required fields in file deploy/olm-catalog/operand-deployment-lifecycle-manager-logging-35211/0.0.1/operand-deployment-lifecycle-manager-logging-35211.v0.0.1.clusterserviceversion.yaml:
	spec.keywords
	spec.maintainers
	spec.provider 
INFO[0084] Created deploy/olm-catalog/operand-deployment-lifecycle-manager-logging-35211/0.0.1/operand-deployment-lifecycle-manager-logging-35211.v0.0.1.clusterserviceversion.yaml 
Running the common required commands for code delivery
make check
make test
?   	github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis	[no test files]
?   	github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator	[no test files]
?   	github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator/v1alpha1	[no test files]
?   	github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller	[no test files]
=== RUN   TestConfigController
--- PASS: TestConfigController (0.00s)
PASS
ok  	github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller/operandconfig	1.588s
=== RUN   TestCatalogController
--- PASS: TestCatalogController (0.00s)
PASS
ok  	github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller/operandregistry	0.963s
=== RUN   TestInitRequest
--- PASS: TestInitRequest (0.00s)
=== RUN   TestDeleteRequest
--- PASS: TestDeleteRequest (0.00s)
PASS
ok  	github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller/operandrequest	0.739s
?   	github.com/IBM/operand-deployment-lifecycle-manager/pkg/util	[no test files]
make build
Building the odlm binary
```

File: deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ spec:
     - name: ibm-monitoring-prometheusext-operator
     - name: ibm-healthcheck-operator
     - name: ibm-monitoring-grafana-operator
+    - name: ibm-elastic-stack-operator
```
Comment (horis233) [inline deploy/crds/operator.ibm.com_v1alpha1_operandrequest_cr.yaml]: Does logging need to be enabled by default?

**Comment by horis233** [comment]
@dian-xh After run `make code-dev`, please commit the csv changes and push it.

**Pull request merged**